build.gradle

```diff
@@ -489,8 +489,11 @@ project(':iceberg-arrow') {
     implementation("org.apache.arrow:arrow-memory-netty") {
       exclude group: 'com.google.code.findbugs', module: 'jsr305'
       exclude group: 'io.netty', module: 'netty-common'
+      exclude group: 'io.netty', module: 'netty-buffer'
     }
 
+    runtimeOnly("io.netty:netty-buffer")
```

Comment:
Just for my understanding. We do need this dep for our Arrow implementation at runtime.
Previously, it came from `arrow-memory-netty` and was shaded in the Spark runtime jar. As far as I see, we kept the shading but also excluded `netty-buffer` from the arrow dependency. What do we shade now?

Reply:
Are we trying to use `netty-buffer` from Spark?

Reply:
We don't use Netty-buffer from Spark. It still comes from `Iceberg-arrow` dependency `arrow-memory-netty`

Reply:
The solution here is to bump it up, so that its version is compatible with both Spark and arrow-memory-netty

Reply:
Can we somehow tell it to pull from `arrow-memory-netty`, so that if `arrow-memory-netty`'s version changes from bumping arrow deps, it will be grabbed. Might be a non-goal though.

Reply:
I did not see the update in `versions.props`. You mean we explicitly define the version in `iceberg-arrow` now? As opposed to consuming whatever `arrow-memory-netty` was bringing?

Reply:
Yes, @aokolnychyi. The intention to make it explicit, instead of leaking from lib `arrow-memory-netty`.
Hi @kbendick, I'm not sure I understand your question. Could you elaborate it?

Reply:
Sorry for the confusion @flyrain. You answered my question via the comment on the `versions.props` file. 👍